src/component/content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+const firstName = ['Yoda','Willy','fool','dilly','billy','Sam', ''];
+
+const lastName =['Sanchez','silly','meow','son','Roger'];
+
+const renderFirst= firstName[Math.floor(Math.random()* firstName.length)];
+    
+const rendeLast= lastName[Math.floor(Math.random()* lastName.length)];
+   
+const renderboth = renderFirst + " " + rendeLast;
+
+
+
+
+
+class Content extends Component{
+    constructor(props){
+        super(props);
+        this.state={};
+
```

Comment:
i think it could also be a good idea to have both names arrays contained either on a separate file or within the class constructor but preferably on a separate file but for now maybe place them here as 
```suggestion
this.firstNames: ['Yoda','Willy','fool','dilly','billy','Sam', ''];
this.lastNames: ['Sanchez','silly','meow','son','Roger'];
```